examples/timeseries.md

```diff
@@ -3,13 +3,15 @@ title: Timeseries
 author: Tom Augspurger
 ---
 
+# Timeseries
```

Comment:
Hmm... Maybe there is no use in viewing this document in GitHub... The Stitch outputs are very importaint in it.